 2018-03-27 22:27:45,557 [P5932/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.9.2 application starting on AMANDEEPCOMPUTE
 2018-03-27 22:27:45,601 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2018-03-27 22:27:45,638 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 36ms)
 2018-03-27 22:27:45,792 [P5932/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2018-03-27 22:27:45,794 [P5932/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2018-03-27 22:27:45,795 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:27:47,152 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1356ms)
 2018-03-27 22:27:47,153 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2018-03-27 22:27:47,154 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 1ms)
 2018-03-27 22:27:47,531 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:27:47,531 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:27:47,532 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2018-03-27 22:27:47,532 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 0ms)
 2018-03-27 22:27:47,532 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:27:47,532 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:27:47,532 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2018-03-27 22:27:47,533 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 1ms)
 2018-03-27 22:27:47,562 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:27:47,562 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:27:47,563 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2018-03-27 22:27:47,563 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 0ms)
 2018-03-27 22:27:47,563 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:27:47,563 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:27:47,564 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2018-03-27 22:27:47,564 [P5932/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 0ms)
 2018-03-27 22:27:50,875 [P5932/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2018-03-27 22:27:51,650 [P5932/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2018-03-27 22:27:52,096 [P5932/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2018-03-27 22:27:52,427 [P5932/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2018-03-27 22:27:52,450 [P5932/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 7079ms)
 2018-03-27 22:27:52,452 [P5932/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalMemberIndexSet to be unlocked since it was left in a locked state
 2018-03-27 22:27:52,454 [P5932/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2018-03-27 22:27:52,454 [P5932/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2018-03-27 22:27:56,746 [P5932/D2/T6] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:57442/umbraco
 2018-03-27 22:27:56,747 [P5932/D2/T6] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57442/umbraco (UmbracoModule request)
 2018-03-27 22:27:57,251 [P5932/D2/T6] INFO  umbraco.content - Load Xml from file...
 2018-03-27 22:27:57,271 [P5932/D2/T6] INFO  umbraco.content - Loaded Xml from file.
 2018-03-27 22:27:57,847 [P5932/D2/T6] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:27:57,848 [P5932/D2/T6] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:27:57,848 [P5932/D2/T6] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2018-03-27 22:27:57,849 [P5932/D2/T6] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 0ms)
 2018-03-27 22:28:57,326 [P5932/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:28:57,326 [P5932/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:28:57,326 [P5932/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2018-03-27 22:28:57,326 [P5932/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 0ms)
 2018-03-27 22:28:57,344 [P5932/D2/T6] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2018-03-26 22:28:57
 2018-03-27 22:29:19,304 [P5932/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:29:19,304 [P5932/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:29:19,304 [P5932/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-03-27 22:29:19,305 [P5932/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2018-03-27 22:29:19,427 [P5932/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:29:19,429 [P5932/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:29:19,429 [P5932/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2018-03-27 22:29:19,429 [P5932/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 0ms)
 2018-03-27 22:29:25,640 [P5932/D2/T6] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username 5066aman@gmail.com from IP address ::1
 2018-03-27 22:29:25,641 [P5932/D2/T6] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: 5066aman@gmail.com logged in from IP address ::1
 2018-03-27 22:29:26,161 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:29:26,161 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:29:26,161 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2018-03-27 22:29:26,161 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 0ms)
 2018-03-27 22:29:26,176 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:29:26,176 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:29:26,176 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-03-27 22:29:26,176 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2018-03-27 22:29:26,176 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:29:26,176 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:29:26,176 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2018-03-27 22:29:26,176 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 0ms)
 2018-03-27 22:29:26,192 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:29:26,192 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:29:26,192 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2018-03-27 22:29:26,192 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 0ms)
 2018-03-27 22:30:51,166 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:30:51,166 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:30:51,166 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Editors.IEditorValidator
 2018-03-27 22:30:51,167 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Editors.IEditorValidator (took 0ms)
 2018-03-27 22:30:51,283 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-03-27 22:30:51,283 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-03-27 22:30:51,283 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2018-03-27 22:30:51,284 [P5932/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 0ms)
 2018-03-27 22:35:14,839 [P5932/D2/T34] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 22:35:14,840 [P5932/D2/T34] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 22:35:14,874 [P5932/D2/T34] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 22:35:14,874 [P5932/D2/T34] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 22:39:35,310 [P5932/D2/T25] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Contact Us' with Id '0' has been published.
 2018-03-27 22:39:39,480 [P5932/D2/T21] INFO  umbraco.content - Save Xml to file...
 2018-03-27 22:39:39,489 [P5932/D2/T21] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 22:39:57,878 [P5932/D2/T41] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Contact Us' with Id '1066' has been published.
 2018-03-27 22:40:01,933 [P5932/D2/T41] INFO  umbraco.content - Save Xml to file...
 2018-03-27 22:40:01,942 [P5932/D2/T41] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 22:40:54,616 [P5932/D2/T13] WARN  Umbraco.Web.UmbracoModule - Status code is 404 yet TrySkipIisCustomErrors is false - IIS will take over.
 2018-03-27 22:41:06,252 [P5932/D2/T25] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Contact Us' with Id '1066' has been published.
 2018-03-27 22:41:10,321 [P5932/D2/T15] INFO  umbraco.content - Save Xml to file...
 2018-03-27 22:41:10,321 [P5932/D2/T15] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 22:42:30,952 [P5932/D2/T13] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 22:42:30,952 [P5932/D2/T13] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 22:42:30,977 [P5932/D2/T13] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 22:42:30,977 [P5932/D2/T13] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 22:50:26,345 [P5932/D2/T46] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 22:50:26,346 [P5932/D2/T46] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 22:50:26,350 [P5932/D2/T48] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 22:50:26,351 [P5932/D2/T48] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 22:51:15,119 [P5932/D2/T41] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-03-27 22:52:22,463 [P5932/D2/T25] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-03-27 22:55:13,955 [P5932/D2/T41] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Contact Us' with Id '1066' has been published.
 2018-03-27 22:55:18,029 [P5932/D2/T21] INFO  umbraco.content - Save Xml to file...
 2018-03-27 22:55:18,036 [P5932/D2/T21] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 22:56:03,270 [P5932/D2/T46] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Contact Us' with Id '1066' has been published.
 2018-03-27 22:56:07,337 [P5932/D2/T50] INFO  umbraco.content - Save Xml to file...
 2018-03-27 22:56:07,345 [P5932/D2/T50] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 22:56:32,317 [P5932/D2/T41] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Homepage' with Id '1056' has been published.
 2018-03-27 22:56:36,358 [P5932/D2/T41] INFO  umbraco.content - Save Xml to file...
 2018-03-27 22:56:36,374 [P5932/D2/T41] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 23:03:42,512 [P5932/D2/T41] WARN  Umbraco.Web.UmbracoModule - Status code is 404 yet TrySkipIisCustomErrors is false - IIS will take over.
 2018-03-27 23:12:29,551 [P5932/D2/T50] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-03-27 23:12:53,034 [P5932/D2/T21] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 23:12:53,034 [P5932/D2/T21] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 23:12:55,389 [P5932/D2/T46] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 23:12:55,389 [P5932/D2/T46] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 23:12:55,391 [P5932/D2/T21] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-03-27 23:13:37,405 [P5932/D2/T46] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 23:13:37,405 [P5932/D2/T46] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 23:13:37,433 [P5932/D2/T48] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 23:13:37,433 [P5932/D2/T48] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 23:20:13,226 [P5932/D2/T25] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Articles' with Id '0' has been published.
 2018-03-27 23:20:17,293 [P5932/D2/T25] INFO  umbraco.content - Save Xml to file...
 2018-03-27 23:20:17,300 [P5932/D2/T25] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 23:21:01,946 [P5932/D2/T13] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Articles' with Id '1074' has been published.
 2018-03-27 23:21:06,009 [P5932/D2/T25] INFO  umbraco.content - Save Xml to file...
 2018-03-27 23:21:06,016 [P5932/D2/T25] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 23:23:52,034 [P5932/D2/T25] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Article1' with Id '0' has been published.
 2018-03-27 23:23:56,113 [P5932/D2/T21] INFO  umbraco.content - Save Xml to file...
 2018-03-27 23:23:56,126 [P5932/D2/T21] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 23:24:40,366 [P5932/D2/T13] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Dilemma of using setState in React LifeCycle Hooks' with Id '1075' has been published.
 2018-03-27 23:24:44,477 [P5932/D2/T25] INFO  umbraco.content - Save Xml to file...
 2018-03-27 23:24:44,485 [P5932/D2/T25] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 23:29:20,539 [P5932/D2/T25] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Lets use composition confidently' with Id '0' has been published.
 2018-03-27 23:29:24,617 [P5932/D2/T25] INFO  umbraco.content - Save Xml to file...
 2018-03-27 23:29:24,628 [P5932/D2/T25] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 23:29:31,641 [P5932/D2/T25] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Let's use composition confidently' with Id '1076' has been published.
 2018-03-27 23:29:35,685 [P5932/D2/T25] INFO  umbraco.content - Save Xml to file...
 2018-03-27 23:29:35,692 [P5932/D2/T25] INFO  umbraco.content - Saved Xml to file.
 2018-03-27 23:37:40,528 [P5932/D2/T48] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field editor
 2018-03-27 23:37:40,528 [P5932/D2/T48] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field hideLabel
 2018-03-27 23:37:40,530 [P5932/D2/T27] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-04-13 21:56:54,220 [P11684/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.10.1 application starting on AMANDEEPCOMPUTE
 2018-04-13 21:56:54,246 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2018-04-13 21:56:54,256 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 10ms)
 2018-04-13 21:56:54,329 [P11684/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2018-04-13 21:56:54,330 [P11684/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2018-04-13 21:56:54,332 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 21:56:54,679 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 348ms)
 2018-04-13 21:56:54,680 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2018-04-13 21:56:54,686 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2018-04-13 21:56:54,701 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 21:56:54,701 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 21:56:54,702 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2018-04-13 21:56:54,707 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 5ms)
 2018-04-13 21:56:54,707 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 21:56:54,708 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 21:56:54,708 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2018-04-13 21:56:54,714 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 5ms)
 2018-04-13 21:56:54,727 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 21:56:54,728 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 21:56:54,728 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2018-04-13 21:56:54,733 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 4ms)
 2018-04-13 21:56:54,733 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 21:56:54,734 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 21:56:54,734 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2018-04-13 21:56:54,739 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 5ms)
 2018-04-13 21:56:56,317 [P11684/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost, InitialCatalog: umbraco_cms. Exception: System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'sa'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:ddd46576-0c76-4db8-821f-01fbcba3321c
Error Number:18456,State:1,Class:14
 2018-04-13 21:56:56,317 [P11684/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-04-13 21:56:56,372 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2018-04-13 21:56:56,381 [P11684/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 9ms)
 2018-04-13 21:56:56,431 [P11684/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost, InitialCatalog: umbraco_cms. Exception: System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'sa'.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:ddd46576-0c76-4db8-821f-01fbcba3321c
Error Number:18456,State:1,Class:14
 2018-04-13 21:56:56,431 [P11684/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-04-13 21:56:57,433 [P11684/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost, InitialCatalog: umbraco_cms. Exception: System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'sa'.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:ddd46576-0c76-4db8-821f-01fbcba3321c
Error Number:18456,State:1,Class:14
 2018-04-13 21:56:57,434 [P11684/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-04-13 21:56:58,435 [P11684/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost, InitialCatalog: umbraco_cms. Exception: System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'sa'.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:ddd46576-0c76-4db8-821f-01fbcba3321c
Error Number:18456,State:1,Class:14
 2018-04-13 21:56:58,436 [P11684/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-04-13 21:56:59,437 [P11684/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost, InitialCatalog: umbraco_cms. Exception: System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'sa'.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:ddd46576-0c76-4db8-821f-01fbcba3321c
Error Number:18456,State:1,Class:14
 2018-04-13 21:56:59,438 [P11684/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-04-13 21:57:00,438 [P11684/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost, InitialCatalog: umbraco_cms. Exception: System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'sa'.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:ddd46576-0c76-4db8-821f-01fbcba3321c
Error Number:18456,State:1,Class:14
 2018-04-13 21:57:00,439 [P11684/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-04-13 21:57:01,444 [P11684/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: localhost, InitialCatalog: umbraco_cms. Exception: System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'sa'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:471daa03-a980-44b9-b47e-e2d70c91f594
Error Number:18456,State:1,Class:14
 2018-04-13 21:57:01,444 [P11684/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-04-13 21:57:01,467 [P11684/D2/T1] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but Umbraco cannot connect to the database.
   at Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   at Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   at Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2018-04-13 21:57:11,520 [P11684/D2/T11] INFO  Umbraco.Core.MainDom - Stopping...
 2018-04-13 21:57:11,521 [P11684/D2/T11] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2018-04-13 21:57:11,827 [P11684/D2/T11] INFO  Umbraco.Web.ExamineStartup - Complete (took 306ms)
 2018-04-13 21:57:11,827 [P11684/D2/T11] INFO  Umbraco.Core.MainDom - Released MainDom.
 2018-04-13 21:57:41,909 [P11684/D2/T7] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: InitializationError

_shutDownMessage=Initialization Error
HostingEnvironment initiated shutdown
HostingEnvironment caused shutdown

_shutDownStack=   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   at System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   at System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   at System.Web.HttpRuntime.AppDomainShutdownTimerCallback(Object state)
   at System.Threading.TimerQueueTimer.CallCallbackInContext(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.TimerQueueTimer.CallCallback()
   at System.Threading.TimerQueueTimer.Fire()
   at System.Threading.TimerQueue.FireNextTimers()
   at System.Threading.TimerQueue.AppDomainTimerCallback()
 2018-04-13 22:04:16,064 [P14096/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.10.1 application starting on AMANDEEPCOMPUTE
 2018-04-13 22:04:16,083 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2018-04-13 22:04:16,096 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 13ms)
 2018-04-13 22:04:16,192 [P14096/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2018-04-13 22:04:16,194 [P14096/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2018-04-13 22:04:16,195 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:16,491 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 295ms)
 2018-04-13 22:04:16,491 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2018-04-13 22:04:16,496 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2018-04-13 22:04:16,514 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:16,515 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:16,515 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2018-04-13 22:04:16,520 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 5ms)
 2018-04-13 22:04:16,521 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:16,521 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:16,521 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2018-04-13 22:04:16,527 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 5ms)
 2018-04-13 22:04:16,540 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:16,540 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:16,540 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2018-04-13 22:04:16,545 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 5ms)
 2018-04-13 22:04:16,546 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:16,546 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:16,546 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2018-04-13 22:04:16,551 [P14096/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 4ms)
 2018-04-13 22:04:18,799 [P14096/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2018-04-13 22:04:19,040 [P14096/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 3046ms)
 2018-04-13 22:04:19,758 [P14096/D2/T8] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:57442/umbraco
 2018-04-13 22:04:19,759 [P14096/D2/T8] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57442/umbraco (UmbracoModule request)
 2018-04-13 22:04:21,760 [P14096/D2/T12] INFO  Umbraco.Core.Configuration.ClientDependencyConfiguration - Updated version number from 528577277 to 1421925920
 2018-04-13 22:04:21,824 [P14096/D2/T12] INFO  Umbraco.Core.MainDom - Stopping...
 2018-04-13 22:04:21,825 [P14096/D2/T12] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2018-04-13 22:04:21,828 [P14096/D2/T12] INFO  Umbraco.Web.ExamineStartup - Complete (took 3ms)
 2018-04-13 22:04:21,829 [P14096/D2/T12] INFO  Umbraco.Core.MainDom - Released MainDom.
 2018-04-13 22:04:23,303 [P14096/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:23,303 [P14096/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:23,303 [P14096/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-04-13 22:04:23,312 [P14096/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 8ms)
 2018-04-13 22:04:27,087 [P14096/D3/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.10.1 application starting on AMANDEEPCOMPUTE
 2018-04-13 22:04:27,105 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2018-04-13 22:04:27,114 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 9ms)
 2018-04-13 22:04:27,167 [P14096/D3/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2018-04-13 22:04:27,168 [P14096/D3/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2018-04-13 22:04:27,170 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:27,218 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 48ms)
 2018-04-13 22:04:27,219 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2018-04-13 22:04:27,219 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 0ms)
 2018-04-13 22:04:27,243 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:27,244 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:27,244 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2018-04-13 22:04:27,244 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 0ms)
 2018-04-13 22:04:27,245 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:27,245 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:27,245 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2018-04-13 22:04:27,246 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 1ms)
 2018-04-13 22:04:27,261 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:27,262 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:27,262 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2018-04-13 22:04:27,262 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 0ms)
 2018-04-13 22:04:27,263 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:27,263 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:27,263 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2018-04-13 22:04:27,264 [P14096/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 0ms)
 2018-04-13 22:04:29,307 [P14096/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2018-04-13 22:04:29,491 [P14096/D3/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 2471ms)
 2018-04-13 22:04:30,203 [P14096/D3/T5] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:57442/umbraco
 2018-04-13 22:04:30,205 [P14096/D3/T5] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57442/umbraco (UmbracoModule request)
 2018-04-13 22:04:30,205 [P14096/D3/T13] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57442/umbraco (UmbracoModule request)
 2018-04-13 22:04:30,994 [P14096/D3/T16] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:30,995 [P14096/D3/T16] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:30,995 [P14096/D3/T16] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2018-04-13 22:04:31,004 [P14096/D3/T16] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 8ms)
 2018-04-13 22:04:31,156 [P14096/D3/T16] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:31,157 [P14096/D3/T16] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:31,157 [P14096/D3/T16] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2018-04-13 22:04:31,162 [P14096/D3/T16] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 5ms)
 2018-04-13 22:04:35,099 [P14096/D3/T13] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username 5066aman@gmail.com from IP address ::1
 2018-04-13 22:04:35,101 [P14096/D3/T13] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: 5066aman@gmail.com logged in from IP address ::1
 2018-04-13 22:04:35,805 [P14096/D3/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:35,805 [P14096/D3/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:35,805 [P14096/D3/T13] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2018-04-13 22:04:35,813 [P14096/D3/T13] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 7ms)
 2018-04-13 22:04:45,404 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Executing installation step: Permissions
 2018-04-13 22:04:45,417 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Step completed (took 12ms)
 2018-04-13 22:04:45,439 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Executing installation step: Upgrade
 2018-04-13 22:04:45,439 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Step completed (took 0ms)
 2018-04-13 22:04:45,484 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Executing installation step: DatabaseInstall
 2018-04-13 22:04:45,486 [P14096/D3/T5] INFO  Umbraco.Core.DatabaseContext - Database configuration status: Started
 2018-04-13 22:04:45,487 [P14096/D3/T5] INFO  Umbraco.Core.DatabaseContext - Database requires upgrade
 2018-04-13 22:04:45,487 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Step completed (took 2ms)
 2018-04-13 22:04:45,504 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Executing installation step: DatabaseUpgrade
 2018-04-13 22:04:45,504 [P14096/D3/T5] INFO  Umbraco.Web.Install.InstallSteps.DatabaseUpgradeStep - Running 'Upgrade' service
 2018-04-13 22:04:45,506 [P14096/D3/T5] INFO  Umbraco.Core.DatabaseContext - Database upgrade started
 2018-04-13 22:04:45,512 [P14096/D3/T5] INFO  Umbraco.Core.Persistence.Migrations.MigrationRunner - Initializing database migrations
 2018-04-13 22:04:45,513 [P14096/D3/T5] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:45,513 [P14096/D3/T5] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:45,514 [P14096/D3/T5] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Persistence.Migrations.IMigration
 2018-04-13 22:04:45,519 [P14096/D3/T5] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Persistence.Migrations.IMigration (took 5ms)
 2018-04-13 22:04:45,535 [P14096/D3/T5] INFO  Umbraco.Core.Persistence.Migrations.MigrationRunner - Added UPGRADE migration 'RenamePreviewFolder' to context
 2018-04-13 22:04:45,584 [P14096/D3/T5] INFO  Umbraco.Core.DatabaseContext - Database configuration status: <p>Upgrade completed!</p>
 2018-04-13 22:04:45,586 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Step completed (took 82ms)
 2018-04-13 22:04:45,598 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Executing installation step: UmbracoVersion
 2018-04-13 22:04:45,607 [P14096/D3/T5] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:45,608 [P14096/D3/T5] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:45,608 [P14096/D3/T5] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2018-04-13 22:04:45,614 [P14096/D3/T5] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 6ms)
 2018-04-13 22:04:45,639 [P14096/D3/T5] INFO  umbraco.content - Load Xml from file...
 2018-04-13 22:04:45,650 [P14096/D3/T5] INFO  umbraco.content - Loaded Xml from file.
 2018-04-13 22:04:45,653 [P14096/D3/T5] INFO  umbraco.content - Loading content from database...
 2018-04-13 22:04:46,268 [P14096/D3/T16] INFO  Umbraco.Core.MainDom - Stopping...
 2018-04-13 22:04:46,889 [P14096/D3/T13] INFO  umbraco.content - Save Xml to file...
 2018-04-13 22:04:46,915 [P14096/D3/T13] INFO  umbraco.content - Saved Xml to file.
 2018-04-13 22:04:46,917 [P14096/D3/T16] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2018-04-13 22:04:46,919 [P14096/D3/T16] INFO  Umbraco.Web.ExamineStartup - Complete (took 1ms)
 2018-04-13 22:04:46,919 [P14096/D3/T16] INFO  Umbraco.Core.MainDom - Released MainDom.
 2018-04-13 22:04:47,094 [P14096/D3/T5] INFO  Umbraco.Web.Install.Controllers.InstallApiController - Step completed (took 1496ms)
 2018-04-13 22:04:51,157 [P14096/D4/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.10.1 application starting on AMANDEEPCOMPUTE
 2018-04-13 22:04:51,175 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2018-04-13 22:04:51,186 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 11ms)
 2018-04-13 22:04:51,243 [P14096/D4/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2018-04-13 22:04:51,245 [P14096/D4/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2018-04-13 22:04:51,246 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:51,307 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 61ms)
 2018-04-13 22:04:51,308 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2018-04-13 22:04:51,309 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 1ms)
 2018-04-13 22:04:51,331 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:51,332 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:51,332 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2018-04-13 22:04:51,332 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 0ms)
 2018-04-13 22:04:51,333 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:51,333 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:51,333 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2018-04-13 22:04:51,334 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 1ms)
 2018-04-13 22:04:51,349 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:51,349 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:51,350 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2018-04-13 22:04:51,350 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 0ms)
 2018-04-13 22:04:51,350 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:51,351 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:51,351 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2018-04-13 22:04:51,351 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 0ms)
 2018-04-13 22:04:51,873 [P14096/D2/T5] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: ConfigurationChange

_shutDownMessage=CONFIG change
HostingEnvironment initiated shutdown
CONFIG change
CONFIG change
CONFIG change
HostingEnvironment caused shutdown

_shutDownStack=   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   at System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   at System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   at System.Web.Configuration.HttpConfigurationSystem.OnConfigurationChanged(Object sender, InternalConfigEventArgs e)
   at System.Configuration.Internal.InternalConfigRoot.OnConfigChanged(InternalConfigEventArgs e)
   at System.Configuration.BaseConfigurationRecord.OnStreamChanged(String streamname)
   at System.Web.Configuration.WebConfigurationHostFileChange.OnFileChanged(Object sender, FileChangeEvent e)
   at System.Web.DirectoryMonitor.FireNotifications()
   at System.Web.Util.WorkItem.CallCallbackWithAssert(WorkItemCallback callback)
   at System.Web.Util.WorkItem.OnQueueUserWorkItemCompletion(Object state)
   at System.Threading.QueueUserWorkItemCallback.WaitCallback_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
 2018-04-13 22:04:54,768 [P14096/D4/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2018-04-13 22:04:54,901 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2018-04-13 22:04:55,546 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 646ms)
 2018-04-13 22:04:55,614 [P14096/D4/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2018-04-13 22:04:55,786 [P14096/D4/T1] WARN  Umbraco.Core.Sync.DatabaseServerMessenger - No last synced Id found, this generally means this is a new server/install. The server will build its caches and indexes, and then adjust its last synced Id to the latest found in the database and maintain cache updates based on that Id.
 2018-04-13 22:04:55,802 [P14096/D4/T1] INFO  umbraco.content - Load Xml from file...
 2018-04-13 22:04:55,803 [P14096/D4/T1] INFO  umbraco.content - Loaded Xml from file.
 2018-04-13 22:04:55,806 [P14096/D4/T1] INFO  umbraco.content - Loading content from database...
 2018-04-13 22:04:56,117 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - Rebuilding index, Provider=InternalMemberIndexer, NodeId=-1
 2018-04-13 22:04:56,188 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - member, Provider=InternalMemberIndexer, NodeId=-1
 2018-04-13 22:04:56,284 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - member, took 95ms, Provider=InternalMemberIndexer, NodeId=-1
 2018-04-13 22:04:56,285 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - Rebuilding index, Provider=InternalIndexer, NodeId=-1
 2018-04-13 22:04:56,315 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - content, Provider=InternalIndexer, NodeId=-1
 2018-04-13 22:04:56,561 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:04:56,561 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:04:56,561 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2018-04-13 22:04:56,562 [P14096/D4/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 0ms)
 2018-04-13 22:04:56,704 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - content, took 388ms, Provider=InternalIndexer, NodeId=-1
 2018-04-13 22:04:56,704 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - media, Provider=InternalIndexer, NodeId=-1
 2018-04-13 22:04:56,816 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - media, took 111ms, Provider=InternalIndexer, NodeId=-1
 2018-04-13 22:04:56,816 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - Rebuilding index, Provider=ExternalIndexer, NodeId=-1
 2018-04-13 22:04:56,833 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - content, Provider=ExternalIndexer, NodeId=-1
 2018-04-13 22:04:56,872 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - content, took 38ms, Provider=ExternalIndexer, NodeId=-1
 2018-04-13 22:04:56,872 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - media, Provider=ExternalIndexer, NodeId=-1
 2018-04-13 22:04:56,876 [P14096/D4/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - media, took 4ms, Provider=ExternalIndexer, NodeId=-1
 2018-04-13 22:04:56,881 [P14096/D4/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2018-04-13 22:04:56,882 [P14096/D4/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2018-04-13 22:04:56,887 [P14096/D4/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 5797ms)
 2018-04-13 22:04:57,129 [P14096/D4/T12] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:57442/umbraco
 2018-04-13 22:04:57,131 [P14096/D4/T12] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57442/umbraco (UmbracoModule request)
 2018-04-13 22:05:00,125 [P14096/D4/T18] INFO  umbraco.content - Save Xml to file...
 2018-04-13 22:05:00,128 [P14096/D4/T18] INFO  umbraco.content - Saved Xml to file.
 2018-04-13 22:05:00,983 [P14096/D4/T5] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:05:00,984 [P14096/D4/T5] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:05:00,984 [P14096/D4/T5] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-04-13 22:05:00,990 [P14096/D4/T5] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 6ms)
 2018-04-13 22:05:01,088 [P14096/D4/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:05:01,088 [P14096/D4/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:05:01,088 [P14096/D4/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2018-04-13 22:05:01,088 [P14096/D4/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 0ms)
 2018-04-13 22:05:01,837 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:05:01,837 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:05:01,837 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2018-04-13 22:05:01,846 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 8ms)
 2018-04-13 22:05:01,858 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:05:01,858 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:05:01,858 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-04-13 22:05:01,859 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2018-04-13 22:05:01,861 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:05:01,861 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:05:01,862 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2018-04-13 22:05:01,868 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 6ms)
 2018-04-13 22:05:01,902 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:05:01,903 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:05:01,903 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2018-04-13 22:05:01,911 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 7ms)
 2018-04-13 22:05:02,556 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-04-13 22:05:02,557 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-04-13 22:05:02,557 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2018-04-13 22:05:02,557 [P14096/D4/T7] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 0ms)
 2018-04-13 22:05:16,940 [P14096/D3/T16] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: ConfigurationChange

_shutDownMessage=CONFIG change
HostingEnvironment initiated shutdown
CONFIG change
HostingEnvironment caused shutdown

_shutDownStack=   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   at System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   at System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   at System.Web.Configuration.HttpConfigurationSystem.OnConfigurationChanged(Object sender, InternalConfigEventArgs e)
   at System.Configuration.Internal.InternalConfigRoot.OnConfigChanged(InternalConfigEventArgs e)
   at System.Configuration.BaseConfigurationRecord.OnStreamChanged(String streamname)
   at System.Web.Configuration.WebConfigurationHostFileChange.OnFileChanged(Object sender, FileChangeEvent e)
   at System.Web.DirectoryMonitor.FireNotifications()
   at System.Web.Util.WorkItem.CallCallbackWithAssert(WorkItemCallback callback)
   at System.Web.Util.WorkItem.OnQueueUserWorkItemCompletion(Object state)
   at System.Threading.QueueUserWorkItemCallback.WaitCallback_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
 2018-04-13 22:05:58,501 [P14096/D4/T12] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2018-04-12 22:05:57
